src/rust/engine/options/src/parse_tests.rs

```diff
@@ -38,7 +40,18 @@ fn check_with_arg<T: PartialEq + Debug>(
 #[test]
 fn test_parse_quoted_string() {
     fn check_str(expected: &str, input: &str) {
-        check!(expected.to_string(), parse_quoted_string(input));
+        // This is slightly convoluted: quoted strings appear as list items,
```

Comment:
This is so I could delete the otherwise dead code `parse_quoted_string()`. It makes sense to specifically test that we handle quoted strings correctly, but now we do so where we actually encounter them - inside lists.